### PR TITLE
CPR-498 Add insight appender for preprod and dev

### PR DESCRIPTION
**Pull request by H-Iwanejko**

### CODE REVIEW

File: src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         <appender name="aiAppender" class="ch.qos.logback.core.helpers.NOPAppender"/>
     </springProfile>
 
-    <springProfile name="prod">
+    <springProfile name="!local">
```
Comment (markreesmoj) [inline src/main/resources/logback-spring.xml:13]: doesn't this conflict with lines 9-11?
[I don't actually think we need this appender at all](https://mojdt.slack.com/archives/C012FNNQ4A3/p1677234268902449?thread_ts=1677233333.667589&cid=C012FNNQ4A3).
Let's take it out and see what happens

**Review by markreesmoj — COMMENTED** [review]
💬

File: src/main/resources/logback-spring.xml (hunk 1 above)
Comment (H-Iwanejko) [reply src/main/resources/logback-spring.xml:9]: Removed 👍

**Review by markreesmoj — APPROVED** [review]
👍

**Pull request merged**